src/components/SelectionList/TransactionListItem.tsx

```diff
@@ -35,145 +47,193 @@ const getTypeIcon = (type?: SearchTransactionType) => {
     }
 };
 
-function TransactionListItem<TItem extends ListItem>({
-    item,
-    isFocused,
-    showTooltip,
-    isDisabled,
-    canSelectMultiple,
-    onSelectRow,
-    onDismissError,
-    shouldPreventDefaultFocusOnSelectRow,
-    onFocus,
-    shouldSyncFocus,
-}: TransactionListItemProps<TItem>) {
-    const transactionItem = item as unknown as TransactionListItemType;
-    const styles = useThemeStyles();
-    const {translate} = useLocalize();
-    const theme = useTheme();
-    const {isLargeScreenWidth} = useWindowDimensions();
-    const StyleUtils = useStyleUtils();
-
-    function getMerchant() {
-        const merchant = TransactionUtils.getMerchant(transactionItem as OnyxEntry<Transaction>);
-        return merchant === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT || merchant === CONST.TRANSACTION.DEFAULT_MERCHANT ? '' : merchant;
-    }
+function arePropsEqual(prevProps: CellProps, nextProps: CellProps) {
+    return prevProps.keyForList === nextProps.keyForList;
+}
 
-    const isFromExpenseReport = transactionItem.reportType === CONST.REPORT.TYPE.EXPENSE;
-    const date = TransactionUtils.getCreated(transactionItem as OnyxEntry<Transaction>, CONST.DATE.MONTH_DAY_ABBR_FORMAT);
-    const amount = TransactionUtils.getAmount(transactionItem as OnyxEntry<Transaction>, isFromExpenseReport);
-    const taxAmount = TransactionUtils.getTaxAmount(transactionItem as OnyxEntry<Transaction>, isFromExpenseReport);
-    const currency = TransactionUtils.getCurrency(transactionItem as OnyxEntry<Transaction>);
-    const description = TransactionUtils.getDescription(transactionItem as OnyxEntry<Transaction>);
-    const merchant = getMerchant();
-    const typeIcon = getTypeIcon(transactionItem.type);
+const DateCell = memo(
+    ({ showTooltip, date, isLargeScreenWidth }: DateCellProps): JSX.Element => {
```

Comment:
We can omit the return type
```suggestion
    ({ showTooltip, date, isLargeScreenWidth }: DateCellProps) => {
```